spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -156,11 +157,23 @@ def get_parser():
     optional.add_argument(
         '-vertfile',
         metavar=Metavar.str,
-        default=os.path.join('.', 'label', 'template', 'PAM50_levels.nii.gz'),
         help=textwrap.dedent("""
             Vertebral labeling file. Only use with flag `-vert`.
 
-            The input and the vertebral labelling file must in the same voxel coordinate system and must match the dimensions between each other.
+            The input and the vertebral labelling file must be in the same voxel coordinate system and must match the dimensions between each other.
+            Example: ./label/template/PAM50_levels.nii.gz
+            This flag will be deprecated in favor of -discfile in the future.
+        """),
+    )
+    optional.add_argument(
+        '-discfile',
+        metavar=Metavar.str,
+        help=textwrap.dedent("""
+            File with disc labels. Only use with flag `-vert`.
```

Comment:
Maybe we could update this description to say something like:

```
File with single-voxel labels identifying the intervertebral discs. Used with `-vert` to aggregate metrics within vertebral levels. Disc labels will be projected onto the spinal cord to identify vertebral levels.
```

That way it's clearer how the `-discfile` interacts with the `-vert` argument?